src/CustomerList/SearchHelper.php

```diff
@@ -138,6 +138,7 @@ public function addListingFilters(Concrete $listing, array $filters, User $admin
                         $segments[] = $segment;
                     }
                 }
+                $handler->getListing()->setGroupBy('o_id', true);
```

Comment:
How about we set groupBy in `CustomerController:buildListing()`? as this will also used by export/download.

Reply:
@aryaantony92 please see